contracts/utils/math/Math.sol

```diff
@@ -553,4 +538,14 @@ library Math {
     function unsignedRoundsUp(Rounding rounding) internal pure returns (bool) {
         return uint8(rounding) % 2 == 1;
     }
+
+    /**
+     * @dev Cast a boolean (false or true) to a uint256 (0 or 1) with no jump.
+     */
+    function boolToUint(bool b) internal pure returns (uint256 u) {
+        /// @solidity memory-safe-assembly
+        assembly {
+            u := b
+        }
+    }
```

Comment:
This looks to me more of a fit into the `SafeCast` library with the `function toUint(bool b)` signature as it already requires a boolean as an argument.

Reply:
I'm totally fine with this, but would it fit the other functions from `SafeCast`? IIUC the main goal of this library is to make dangerous operations and throw in case of an overflow In `toUint256(bool)` there's no overflow possible, there's nothing "safe" about this `SafeCast` function.

Reply:
Yeah that's right, I don't have the last word but I feel confident that `SafeCast` is a better fit than `Math`.

We might consider having these unconventional casts in a different library but that's a broader discussion.